MetaPhase-Consulting/State-TalentMAP
# Tm 6874 hide assignments separations link

[Ticket](https://metaphase.atlassian.net/jira/software/c/projects/TM/boards/1?assignee=712020%3A3d9639a5-439c-4e2d-9bce-37b6924384d2&selectedIssue=TM-6874)

Description: If a user doesn't have fsbid_admin and/or superuser role, they should not be able to see the "Add New Assignment/Separation" link on the Maintain Assignments page. Scroll to the bottom for testing instructions (also available on the ticket itself)

With super user role present:

<img width="456" alt="superuser role" src="https://github.com/user-attachments/assets/1a1028f7-1390-4cf0-886d-3d4b9a63c373" />

The link is also present:

<img width="1116" alt="link available" src="https://github.com/user-attachments/assets/0f28bc38-c83b-4e7e-a41d-94c1552fd32d" />

Getting my superuser and/or fsbid_admin role taken away:

<img width="472" alt="current roles (no admin:su)" src="https://github.com/user-attachments/assets/bb5399a0-3489-4848-8a91-e9bff3f4506a" />

No link shows up now:

<img width="1120" alt="no current assignment link" src="https://github.com/user-attachments/assets/e05f7f13-c8fc-4af9-805b-0e642f4ab22f" />

TESTING INSTRUCTIONS:

1. Check if admin/superuser role is currently assigned to you.
2. If admin and/or superuser role present, navigate to Maintain Assignments like for an AO/CDO and check for the presence of "Add New Assignment/Separations" link on the top right of the Assignment History page:

- Dashboard -> AO/CDO -> Employees and Panel Agendas -> Pick user card -> Assignment History section -> Maintain Assignments link -> Check for the presence of "Add New Assignment/Separation" link on top right.

3. Ask Morgan to take away your admin/superuser role(s) to test the opposite in step 4.

4. If admin and/or superuser role NOT present, navigate to Maintain Assignments like for an AO/CDO and "Add New Assignment/Separations" link on the top right should NOT be present. Only "Add Notification" and "Add Memo" links will be available:

- Dashboard -> AO/CDO -> Employees and Panel Agendas -> Pick user card -> Assignment History section -> Maintain Assignments link -> There should be no "Add New Assignment/Separation" link

NOTE - if you didn't have admin/superuser and tested out Step 4, please ask Morgan to give you those roles so Step 3 can be tested.
